data/hai/unfettered 2 war preparations.txt

```diff
@@ -1056,9 +1057,12 @@ mission "<uhai> 15: Modified Beetle"
 					"outfit (all): Ionic Turret Prototype" > "turrets owned"
 			branch normal
 				not "unfettered rep: Mera"
-			`	"Since this was partly a test-run, what suggestions would you make to improve the design, before I finally give this ship to you?"`
+			`	"Since this was partly a test-run, what suggestions would you make to improve the ship, so I can make changes to its outfits, before I finally give this ship to you?"`
+			label suggestions
 			choice
 				`	"The ship is fine, but I could use an extra outfit to modify the loadout myself."`
+				`	"Can you explain to me why you named the ship Hera?"`
```

Comment:
It looks like this choice doesn't hide the option after it's taken? Not sure it's necessary, but I think the text flow from this option would work better that way.